test/parallel/test-cluster-worker-disconnect.js

```diff
@@ -40,7 +40,8 @@ if (cluster.isWorker) {
 
   // Disconnect worker when it is ready
   worker.once('listening', common.mustCall(() => {
-    worker.disconnect();
+    const w = worker.disconnect();
+    assert.strictEqual(worker, w, 'did not return a reference');
```

Comment:
You've already made changes to the test file, that's good enough IMHO. I believe these changes here can be removed.